projects/FHC-AtmelStudio/Fussl-01/Fussl-01/f_range_int.h

```diff
@@ -0,0 +1,153 @@
+/*
+* f_range_int.h
+*
+* Created: 13.03.2018 17:52:31
+* @author Maximilian Starke
+*/
```

Comment:
Check what doxygen is doing with this leading comment.

Reply:
Doxygen's html pages contain a list of files.
There one can see the general description of a file